apps/docs/pages/docs/api-reference/fields/external.mdx

```diff
@@ -611,4 +612,54 @@ const config = {
 }}
 />
 
+### `renderFooter(props)`
```

Comment:
As with my other comment, can you move this above the `### showSearch` heading to make it alphabetical?